certification/internal/utils/container/helper.go

```diff
@@ -97,3 +98,24 @@ func RunInsideContainerFS(podmanEngine cli.PodmanEngine, image string, container
 
 	return containerFn(report.MountDir)
 }
+
+func GenerateBundleHash(podmanEngine cli.PodmanEngine, image string) (string, error) {
+	hashCmd := `find . -not -name "Dockerfile" -type f -printf '%f\t%p\n' | sort -V -k1 | cut -d$'\t' -f2 | tr '\n' '\0' | xargs -r0 -I {} md5sum "{}"` // >> $HOME/hashes.txt`
```

Comment:
It would be great to define this as a const on top and not recreate it every time this function is called.

Reply:
It's only called one time. And since it is basically a constant, the compiler probably optimizes for that anyway.